smtp_connector.py

```diff
@@ -72,17 +71,18 @@ def initialize(self):
         self._state = self.load_state()
         if not isinstance(self._state, dict):
             self.debug_print("Resetting the state file with the default format")
-            self._state = {"app_version": self.get_app_json().get("app_version")}
+            self._reset_state()
             return self.set_status(phantom.APP_ERROR, SMTP_STATE_FILE_CORRUPT_ERROR)
 
         # action_result = self.add_action_result(ActionResult())
 
-        self.set_validator('email', self._validate_email)
+        # skipping the validation for email
```

Comment:
Why not remove this completely if the validation function has been removed?

Reply:
Custom email validation was removed, but since the contains still has an email, it gets validated using predefined rules. To bypass this, set_validator was overridden and set to true to remove the validation.